projects/netgrif-components-core/src/assets/i18n/de.json

```diff
@@ -232,6 +232,8 @@
             "phone": "Der eingegebene Text muss ein Telefonnummer sein",
             "email": "Der eingegebene Text muss ein E-Mail sein",
             "dateRange": "Das eingegebene Datum muss zwischen {{left}} und {{right}} liegen",
+            "datePast": "Das eingegebene Datum muss bis sein {{right}}",
+            "dateFuture": "Das eingegebene Datum muss von sein {{left}}",
```

Comment:
`ab` instead of `von`.

Both messages should have the verb (`sein`) at the end of the sentance `... {{right/left}} sein`

Reply:
done